subprojects/gst-plugins-bad/sys/msdk/gstmsdkvpp.c

```diff
@@ -921,7 +921,7 @@ gst_msdkvpp_transform (GstBaseTransform * trans, GstBuffer * inbuf,
     free_msdk_surface (in_surface);
     return GST_FLOW_ERROR;
   }
-  locked_by_others = ! !in_surface->surface->Data.Locked;
+  locked_by_others = !!in_surface->surface->Data.Locked;
```

Comment:
I remembered this fix would trigger a gst-indent fail in GStreamer upstream pipeline check. Please notice this when you are preparing for upstream.